.github/workflows/issue.yml

```diff
@@ -1,6 +1,9 @@
 name: New issue
 
 on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Do we want to create tickets for PRs?

Reply:
This is just a temporary change for ease of testing